.rubocop.yml

```diff
@@ -0,0 +1,58 @@
+AllCops:
+  Exclude:
+    - "README.md"
+    - "Guardfile"
+    - "Rakefile"
+
+  DisplayCopNames: true
+
+Layout/LineLength:
+  Max: 120
+Metrics/MethodLength:
+  Max: 20
+Metrics/AbcSize:
+  Max: 50
+Metrics/ClassLength:
+  Max: 150
+Metrics/BlockLength:
+  ExcludedMethods: ["describe"]
+  Max: 30
+Metrics/ModuleLength:
+  Enabled: false
+Metrics/CyclomaticComplexity:
+  Enabled: false
+Metrics/PerceivedComplexity:
+  Enabled: false
+
+Style/CaseEquality:
+  Enabled: false
```

Comment:
You should not alter the default configured file provided from Microverse. If you think that any of the rules are needed to be disabled, Please open a Merge Request in this [link](https://github.com/microverseinc/linters-config/tree/master/ruby)